src/utils/constants.ts

```diff
@@ -205,6 +209,9 @@ export const PERMIT2: Record<number, Address> = {
 export const AUTHORIZER: Record<number, Address> = {
     [ChainId.SEPOLIA]: '0xa331d84ec860bf466b4cdccfb4ac09a1b43f3ae6',
 };
+
+export const ADMIN_OF_AUTHORIZER = '0x171C0fF5943CE5f133130436A29bF61E26516003'; // is this EoA chain agnostic?
```

Comment:
From what I can tell, this EoA i capable of granting different roles, so maybe this is better? Are there multiple admins? Idk that much about OZ authorizer

https://sepolia.etherscan.io/address/0x171C0fF5943CE5f133130436A29bF61E26516003

Reply:
I'm not sure of details here either. Maybe Mk will have an idea.